environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/AttachedFreeIpaRequest.java

```diff
@@ -17,6 +19,10 @@
     @ApiModelProperty(value = EnvironmentModelDescription.FREEIPA_INSTANCE_COUNT_BY_GROUP)
     private Integer instanceCountByGroup;
 
+    @Valid
```

Comment:
please add @Valid to `private AttachedFreeIpaRequest freeIpa;` line 72 the EnvironmentRequest class